repo/packages/M/minio/6/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "Minio",
```

Comment:
change `Minio` -> `minio`
Changing the name here creates a different package in Universe.